docs/dashboard/dashboard.rst

```diff
@@ -0,0 +1,271 @@
+
+   A newly created environment
+
+This environment is currently empty because the model has not been compiled yet.
+We can use the ``Recompile`` button to do this.
+This will clone the repository if it hadn't been already and then compile the current environment.
+There is also an extra option for the recompile, which is ``Update project & Recompile``.
+This will pull in any new commits and then compile the model.
+
+Once the compile has succeeded, inmanta will automatically deploy the model and display the currently deployed model in the portal.
+
+Using the compile report button we can diagnose problems if our compile failed.
+
+.. figure:: /dashboard/images/img_6.png
+   :width: 100%
+   :align: center
+   :alt: A compile report
+
+   A compile report
+
+You can click the arrow icon next to any item to expand it and see the output of the executed command.
+
+Next, we have the ``Force deploy`` and ``Force repair`` buttons.
+Those are similar in function, and can be confusing to new users:
+
+- The ``Force deploy`` button will go through *Every* resource and redeploy the resource.
+- The ``Force repair`` button by contrast, will only go through resources that are currently not in a deployed state.
+
+Finaly we have the ``Decommission``, ``Edit``, ``Clone`` and ``Clear`` buttons, found under the Decommission dropdown menu:
+
+- Decommission: pushes a model that purges all resources deployed by the model.
+- Edit: change the configuration of the environment, such as the git repo url or what branch to use.
+- Clone: create a new environment using the same git repo and branch
+- Clear: Clears the environment. This will remove all versions and compilations. It does not decommission the currently deployed model.
+
+.. note::
+    When using clear followed by a recompile, the version number will be incremented as if the previous version are still there, but these versions will no longer be present.
+
+The Version overview
+--------------------
+
+Below the ``Portal`` we have the ``Versions``.
+This will take us to an overview of all previously compiled versions of the model and their state.
+Do note that a version is created for every compile and this is not tied to the model being updated in git.
+
+Each version has 4 buttons on the right to interact with it:
+
+.. figure:: /dashboard/images/img_7.png
+   :width: 100%
+   :align: center
+   :alt: Version buttons
+
+They are, in order from left to right:
+
+- Perform dry run: the orchestrator will go through all resources in the model and compare their current state to their desired state. This can be usefull to doublecheck what effect the deployment of a version might have on your current environment.
+- Dry run report: will take you to the report of the last performed dry run, without performing a new one.
+- Release version: deploys the specific version of said model. For example, this can be used to deploy an older version of a model or to skip several versions.
```

Comment:
The button releases the model if auto-deploy is set to false, but nowadays auto-deploy is set to true by default. In that case it's redundant an will be grayed out. This button cannot be used to deploy an older version of the model.

Reply:
Updated